pkg/commands/apply.go

```diff
@@ -96,7 +96,7 @@ func addApply(topLevel *cobra.Command) {
 			if err != nil {
 				return fmt.Errorf("error creating builder: %w", err)
 			}
-			publisher, err := makePublisher(po)
+			publisher, err := makePublisher(po, bo.Envs)
```

Comment:
Passing a build option to `makePublisher` is a bit of a smell... Should it be a `publish.Option`? 🤔

Reply:
Yeah, I've updated the code like this 🙋🏻‍♂️